api.md

```diff
@@ -1822,6 +1823,33 @@ ORDER BY day;
 
 ## Utilities/Statistics [](utilities)
 
+### timescaledb_information.data_node [](#timescaledb_information-datanode)
+
+Get information on data nodes.
```

Comment:
It might be worth mentioning somewhere in this description that the 'num_dist_tables' and 'num_dist_chunks' are only available if the node is up.

Reply:
Given that content on this page will include both single-node and clustering, you might want to clarify in top of this function (and all such function) that it corresopnds to timescaledb running in a clustered configuration, consistenting of a front-end access node and back-end data nodes.

Reply:
Added the mention about those metrics only being available when a node is up in the actual table. I will add a sentence in the first part of the function detailing that